cmd/tarmak/e2e/sonobuoy_test.go

```diff
@@ -0,0 +1,171 @@
+// Copyright Jetstack Ltd. See LICENSE for details.
+package e2e_test
+
+import (
+	"compress/gzip"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/pkg/archive"
+)
+
+var mode = "Quick"
+
+type SonobuoyInstance struct {
+	t            *testing.T
+	binPath      string // bin path to tarmak binary
+	mode         string // quick or conformance
+	retrievePath string
+	kubeconfig   string
+}
+
+func NewSonobuoyInstance(t *testing.T, environment, kubeconfig string) *SonobuoyInstance {
+	si := &SonobuoyInstance{
+		t:       t,
+		binPath: "../../../bin/sonobuoy",
```

Comment:
Not too sure where is this supposed to come from?

Did you want to add it to the Makefile